### PR TITLE
Add link to Debian package to README.md

### PR DESCRIPTION
**Pull request by antonsoroko**
Added link to Debian package and mention its maintainer. Also i removed mention about ubuntu version since they always changes b/c of "end of life/support". And i removed mention about kububntu since there are many other flavours of ubuntu like xubuntu.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,19 +45,21 @@ available from the standard repositories for your distribution, you can try
...
-- Ubuntu/Kubuntu package in [Solaar stable ppa][ppa2]
+- Ubuntu/Kubuntu package in [Solaar stable ppa][ppa stable]
 - NixOS Flake package in [Svenum/Solaar-Flake][nix flake]
 
 Solaar is available from some other repositories
 but may be several versions behind the current version:
 
-- for Ubuntu/Kubuntu 16.04+: the solaar package from [universe repository][universe repository]
+- a [Debian package][debian], courtesy of Stephen Kitt
+- an Ubuntu package is available from [universe repository][ubuntu universe repository]
```
Comment (pfps) [inline README.md:55]:
```suggestion
- a Ubuntu package is available from [universe repository][ubuntu universe repository]
```

**Review by pfps — CHANGES_REQUESTED** [review]
Grammar nit.

File: README.md (hunk 1 above)
Comment (antonsoroko) [reply README.md:55]: @pfps 
Wikipedia also uses `an`:
image

**Comment by pfps** [comment]
Cute but not necessary here.

**Comment by antonsoroko** [comment]
ok. i applied your changes.

**Pull request merged**